### ISSUE TITLE
Cannot backup paths with spaces

### ISSUE THREAD

**Issue by djmaze** [djmaze/resticker#122]
### Discussed in https://github.com/djmaze/resticker/discussions/114

<div type='discussions-op-text'>

<sup>Originally posted by **nonsavant** December 29, 2021</sup>
Tried every possible escaping mechanism that I could think of and I cannot get RESTIC_BACKUP_SOURCES to accept an argument including a space.

`RESTIC_BACKUP_SOURCES: data/my folder`
`RESTIC_BACKUP_SOURCES: "data/my folder"`
`RESTIC_BACKUP_SOURCES: "'data/my folder'"`
`RESTIC_BACKUP_SOURCES: data/my\ folder`
`RESTIC_BACKUP_SOURCES: "data/my\ folder"`

No matter what I try, restic treats it as two separate backup sources, always delimiting on the space. E.g.

```
/data/my does not exist, skipping
folder does not exist, skipping
```
</div>

**Comment by djmaze** [comment]
@nonsavant Sorry for the late response. I might have a fix in https://github.com/djmaze/resticker/pull/120. Will test now.

**Comment by DJJo14** [comment]
Is this the same case with spaces in RESTIC_BACKUP_ARGS?

because i get a unable to find Music from this :
`--exclude="99-Backup/My Music"`

**Comment by djmaze** [comment]
Should be fixed in #120 now as well, but you will have to use the form `--exclude "99-Backup/My Music"` instead.
